content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/manual.md

```diff
@@ -162,7 +162,7 @@ corda-cli.sh topic -b kafka-1.example.com -k config.properties \
 {{% /tab %}}
 {{% tab name="Windows" %}}
 ```shell
-corda-cli.cmd topic -b kafka-1.example.com -k config.properties create -r 3 -p 10 script -f create.cmd -c 6
+corda-cli.cmd topic -b kafka-1.example.com -k config.properties create -r 3 -p 10 script -f create.sh -c 6
```

Comment:
@nadinequinr3 - this doesn't make sense to me - using create.sh on the windows platform.
Unless, the create.sh script is passed to a linux runnable image or something...

Reply:
It is correct from a docs perspective in that the `topic script` CLI plugin is currently only capable of generating a bash script. So, yes, the generated script would need to run under WSL, in a Linux container, on another machine... I'll raise with PM what we want to do about Windows here.